### PR TITLE
chore: update to 4.31

### PR DESCRIPTION
**Pull request by andygup**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,172 +1,52 @@
 # Change log
 
-The `next` version of 4.30 is now available. Planned release date is June 2024.
+The `next` version of 4.31 is now available. Planned release date is October 2024.
 
 ![Current build version](https://img.shields.io/npm/v/@arcgis/core/next?label=Current%20build)
 
 ## Components updates
```
Comment (bsvensson) [inline CHANGELOG.md:7]:
```suggestion
## Component packages

- Chart Components might come out of beta at 4.31.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,172 +1,52 @@
...
-- Customizable graphics colors
-- Following options (sensor, frame, video)
-- Adjustable speed and quality
-- Access to frame metadata
-
 ## Layer updates
 
-TBD
-
 ## Widget Updates
```
Comment (bsvensson) [inline CHANGELOG.md:11]:
```suggestion
## Component and widget updates
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,172 +1,52 @@
...
-- BUG-000166346: Fixed an issue where polygon features with newlines in the label expression did not display correctly.
-- BUG-000167231: Fixed an issue where a CIMSymbol with CIMGeometricEffectDashes was having performance issues at higher scales.
-- [Esri Community - 1403039](https://community.esri.com/t5/arcgis-javascript-maps-sdk-questions/4-29-breaking-change-mapview-goto-w-rotation/m-p/1403039): Fixed an issue where MapView.goTo would zoom out when only the rotation was set in the target.
-- [Esri Community - 1407481](https://community.esri.com/t5/arcgis-javascript-maps-sdk-questions/first-calls-to-sendmessagetoclient-not-used/m-p/1407481): Fixed an issue where the first call to StreamLayer.sendMessageToSocket is not applied on initial load.
-- [Esri Community - 1413086](https://community.esri.com/t5/arcgis-javascript-maps-sdk-questions/4-29-memory-leak-on-streamlayer/m-p/1413086): Fixed an issue where the client-side `StreamLayer` was causing a memory leak.
-- Fixed an issue where setting popupEnabled to false was not properly honored for LayerSearchSource.
-- The Features widget is out of beta.
-- VideoElement is now supported on iPhones.
-
 ## Deprecations
```
Comment (bsvensson) [inline CHANGELOG.md:13]:
```suggestion
# Breaking Changes

Please refer to the [Breaking changes](https://developers.arcgis.com/javascript/latest/breaking-changes/) guide topic for a complete list of breaking changes across all releases of the 4.x.

## Deprecations
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,172 +1,58 @@
...
-
-- The AMD npm package [`arcgis-js-api`](https://www.npmjs.com/package/arcgis-js-api) is deprecated at version 4.29. This is related to dropping local build support for legacy Dojo 1 and RequireJS. Use [@arcgis/core](https://developers.arcgis.com/javascript/latest/es-modules/) ES modules or plan on moving to [components (beta)](https://developers.arcgis.com/javascript/latest/components/) instead.
-- The npm package [`@arcgis/cli`](https://www.npmjs.com/package/@arcgis/cli) is deprecated at version 4.29. Plan on moving to [components (beta)](https://developers.arcgis.com/javascript/latest/components/) and Calcite.
-- The [`esri-loader`](https://github.com/Esri/esri-loader/blob/master/README.md) library is deprecated at version 4.29. Use [@arcgis/core](https://developers.arcgis.com/javascript/latest/es-modules/) ES modules or plan on moving to [components (beta)](https://developers.arcgis.com/javascript/latest/components/) instead.
```
Comment (annelfitz) [inline CHANGELOG.md:135]: Shouldn't these still be listed here?

File: CHANGELOG.md [hunk 5]
```diff
@@ -176,31 +62,31 @@ The following are deprecated and will be removed in a future release. For anythi
 - ImageryTileLayer.rasterInfo deprecated since 4.29. Use serviceRasterInfo instead.
 - LayerList.multipleSelectionEnabled deprecated since 4.29. Use selectionMode instead.
 - LayerList.selectionEnabled deprecated since 4.29. Use selectionMode and dragEnabled instead.
-- Lighting.clone deprecated since version 4.24
-- Lighting deprecated since version 4.24. Use SunLighting instead.
+- ListItemPanel.className deprecated since version 4.30. Use icon
+- ListItemPanel.className deprecated since version 4.30. Use icon
```
Comment (annelfitz) [inline CHANGELOG.md:66]: This is a duplicate (I realize there is also a duplicate in the release notes -- but they are linking to different classes.. LayerList vs TableList)

**Review by annelfitz — APPROVED** [review]
LGTM!

File: CHANGELOG.md (hunk 4 above)
Comment (andygup) [reply CHANGELOG.md:135]: Thanks for the reminder. I needed to move them from Deprecations to the Breaking Changes section. Done.

File: CHANGELOG.md (hunk 5 above)
Comment (andygup) [reply CHANGELOG.md]: Done.

**Comment by annelfitz** [comment]
thanks @andygup !

**Review by bsvensson — APPROVED** [review]

**Pull request merged**